script.js

```diff
@@ -30,3 +30,152 @@ contactLink.addEventListener('click', () => {
+    featuredImage: 'images/snapsnap.png',
+    technologies: ['Ruby on rails', 'css', 'Javascript', 'html'],
+    liveVersion: '',
+    linkSource: '',
+  },
+];
+
+const cardsPortfolio = document.querySelector('.cards-portfolio');
+let portfolioSection = '';
+portfolioObj.forEach((item, index) => {
+  let technologies = '';
+  item.technologies.forEach((tech) => {
+    technologies += `<li>
+    <a href='#'
+      ><button class='btn-hover btn-active btn-pressed'>
+        ${tech}
+      </button></a
+    >
+  </li>`;
+  });
+
+  const newItem = `
+  <div class='project'>
+    <div class='image-back'></div>
+    <div class='project-footer'>
+      <div class='project-title-back'>
+        <h4>${item.name}</h4>
+      </div>
+      <ul class='project-buttons-back'>
+        ${technologies}
+      </ul>
+      <button id='${index}' class='works-button btn-hover btn-active btn-pressed'>
+        See Project
+      </button>
+    </div>
+  </div>
+  `;
+  portfolioSection += newItem;
+});
+
+cardsPortfolio.innerHTML += portfolioSection;
+
+const worksButtons = document.querySelectorAll('.works-button');
+const popupContainer = document.querySelector('.popup-container');
+const popupMenu = document.querySelector('.popup-menu');
+
+worksButtons.forEach((button) => {
+  button.addEventListener('click', (e) => {
+    const portfolioItem = portfolioObj[Number(e.target.id)];
+
+    let technologies = '';
+    portfolioItem.technologies.forEach((tech) => {
+      technologies += `<li>
+      <button class='btn-hover btn-active btn-pressed'>
+        ${tech}
+      </button>
+    </li>`;
+    });
+
+    popupMenu.innerHTML = `
+      <i class='fa fa-close close-popup' id="close-popup"></i>
+      <div class="close-popup-image">
+        <i class='fa fa-close close-popup'></i>
+        <img alt='' src='${portfolioItem.featuredImage}' />
+      </div>
+      <div class="popup-header">
+        <h2>${portfolioItem.name}</h2>      
+        <div class='popup-buttons-desktop'>
+          <button>
+            See Live
+            <i class='fa-solid fa-arrow-up-right-from-square'></i>
+          </button>
+          <button>
+            See Source
+            <i class='fa-brands fa-github'></i>
+          </button>
+        </div>
+      </div>
+      <ul>
+        ${technologies}
+      </ul>
+      <p>
+        ${portfolioItem.description}
+      </p>
+      <div class='popup-buttons'>
+        <button>
+          Live
+          <i class='fa-solid fa-arrow-up-right-from-square'></i>
+        </button>
+        <button>
+          Source
+          <i class='fa-brands fa-github'></i>
+        </button>
+      </div>
+    `;
+    popupContainer.style.display = 'flex';
+    const closePopupButtons = document.querySelectorAll('.close-popup');
+    closePopupButtons.forEach((button) => {
+      button.addEventListener('click', () => {
+        popupContainer.style.display = 'none';
+      });
+    });
+  });
+});
```

Comment:
- [ ] Kindly make sure there is a close button for the modal in the desktop view, this is currently missing please add it back as I see it was ok in the previous review, this will create a better user experience 👍🏿 

- [ ] Please take note of the button appearance, just as the previous reviewer requested, kindly remove the border outlines for all buttons in the popup
- [ ] It would be better to adjust the content of the `see live and see source` buttons to look like there were in the previous review, currently, the icon is not aligned with the text

Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![Template 3-detail](https://user-images.githubusercontent.com/69638013/216061886-29f0e7c1-1b42-4860-9b89-373a1c243336.png)  | ![Screenshot_688](https://user-images.githubusercontent.com/69638013/216060601-958cd16a-ab3a-4241-b524-c3508621f86d.png)